Vizzuality/gfw
# Fix/bug fixes for bugs

Hotfixes for:
- HMR not completely covered for state() cases
- fill on new svg preventing white fill on mobile
- analysis panel crashing app in mobile due to infinite rerender from datasets selectors

To test:
- load app in mobile view and make analysis
- check mobile icons for datasets
- HMR shouldn't crash app when viewing analysis and making a change
